spec/suites/layer/marker/Marker.DragSpec.js

```diff
@@ -102,7 +103,7 @@ describe("Marker.Drag", function () {
 				onStop: function () {
 					var center = map.getCenter();
 					expect(center.lat).to.be(0);
-					expect(center.lng).to.be.within(10, 30);
+					expect(center.lng).to.be.within(0, 30);
```

Comment:
Why change this?

Reply:
Actually I don't remember anything literally anything about that, I forget I opened this PR :D